dash_bio/Ideogram.py

```diff
@@ -130,21 +135,21 @@ class Ideogram(Component):
 One of 450, 550, or 850.
 - filterable (boolean; optional): Whether annotations should be filterable or not.
 - orientation (a value equal to: 'vertical', 'horizontal'; optional): The orientation of chromosomes on the page.
-- ploidy (number; optional): The ploidy - number of chromosomes to depict for each chromosome set.
-- ploidyDesc (list; optional): Description of ploidy in each chromosome set in terms of ancestry composition.
+- ploidy (number; default 1): The ploidy - number of chromosomes to depict for each chromosome set.
+- ploidyDesc (list of dicts; optional): Description of ploidy in each chromosome set in terms of ancestry composition.
 - ancestors (dict; optional): A map associating ancestor labels to colors. Used to color
 chromosomes from different ancestors in polyploid genomes.
-- rangeSet (list; optional): List of objects describing segments of recombination among chromosomes in a chromosome set.
-- rotatable (boolean; optional): Whether chromosomes are rotatable on click.
+- rangeSet (list of dicts; optional): List of objects describing segments of recombination among chromosomes in a chromosome set.
+- rotatable (boolean; default True): Whether chromosomes are rotatable on click.
 - rotated (boolean; optional): Dash callback that returns true if rotated, and false if not.
 - sex (a value equal to: 'male', 'female'; optional): Useful for omitting chromosome Y in female animals.
 Currently only supported for organisms that use XY sex-determination.
-- showChromosomeLabels (boolean; optional): Whether to show chromosome labels, e.g., 1, 2, 3, X, Y.
-- showBandLabels (boolean; optional): Whether to show cytogenetic band labels, e.g., 1q21
-- showFullyBanded (boolean; optional): Whether to show fully banded chromosomes for genomes that have sufficient data. Useful for
+- showChromosomeLabels (boolean; default True): Whether to show chromosome labels, e.g., 1, 2, 3, X, Y.
+- showBandLabels (boolean; default False): Whether to show cytogenetic band labels, e.g., 1q21
```

Comment:
```suggestion
- showBandLabels (boolean; default False): Whether to show cytogenetic band labels, e.g., 1q21.
```

Reply:
Could you make these suggestions in the source file (`src/lib/components/Ideogram.react.js`) instead? The Python files are generated from the JS ones.

Reply:
Right, I thought of it afterwards.

Reply:
Actually, `src/lib/components/Ideogram.react.js` isn't part of the changesets included in this PR... How come?

Reply:
Because that file didn't change at all. The reason the Python files changed is due to the updated docstring formatting introduced in https://github.com/plotly/dash/pull/782.

Reply:
Oh, ok! I see that a lot is happening in this PR... not just the migration of the test suite, which is already something substantial... Anyway, I'll remove these suggestions, they will go into a separate PR.